### DISCUSSION TITLE
[HW2] Question about recall in 2.3

### DISCUSSION THREAD

**Discussion by Hyungi-Lee** [seominjoon/kaist-ai605#102] [Q&A] [answered]
Hi professor and TAs.
I have a question about measuring recall rate in HW2 problems.
In order to measure the recall rate, I think we need ground truth for SQuAD validation set.
How can we get the ground truth of SQuAD validation set?

**Comment by seominjoon** [comment, marked as answer]
The ground truth is already in the data (i.e. each question should be able to retrieve the context document that it is contained in among all context documents). Could you be please more specific what you mean by the ground truth?

**Reply by yuneg11** [reply]
I found that the validation set contains lots of duplicated contexts (some of them have more than 10 duplicates).
So, when we retrieve 10 documents with a question, the true index may not appear in the result because duplicates have the same score.
Should we remove the duplicates?

**Reply by seominjoon** [reply]
Thanks for pointing it out. Yes, please remove the duplicate. An alternative way is that you can evaluate the retrieval to be correct if the retrieved documents are identical (content-wise).
